### DISCUSSION TITLE
update-consumer-pom system scoped dependencies warning

### DISCUSSION THREAD

**Discussion by lppedd** [eclipse-tycho/tycho#2056] [Q&A] [answered]
I'm getting a lot of these log warnings:

```
[INFO] --- tycho-packaging-plugin:4.0.0-SNAPSHOT:update-consumer-pom (default-update-consumer-pom) @ com.ibm.pdtools.common.component.core ---
[WARNING] 0 system scoped dependencies were mapped to maven artifacts, 111 were skipped
```

Should I be worried about something, or is this somewhat expected?

**Comment by laeubi** [comment, marked as answer]
> Should I be worried about something, or is this somewhat expected?

It depends, in this case it means your consumer pom will most likely not be complete.

**Reply by lppedd** [reply]
Ok! I'll debug to check why they're not mapped.
